src/services/createExternalRoutes.spec.ts

```diff
@@ -17,7 +16,6 @@ test('when given external route props with a host and children return host for e
   const [routeA, routeB] = createExternalRoutes([
     {
       host: 'https://kitbag.dev',
-      path: '/',
```

Comment:
Isn't needed for this test. But routeB as a url would end up being https//kitbag.devchild right?

Reply:
ah yeah, I'll fix